README.md

```diff
@@ -35,9 +35,9 @@ Please help us out if you think the roundup could be better! [Open an issue!](ht
 
 ## Process for writing 'IPFS Weekly' roundup
 
-- Open an issue for the new roundup, and fill it with the draft
-- Wait 24 hours for comments; digest and work them into the draft
-- Put the draft into a `.md` file inside of the `drafts` folder
+- Open a Pull Request to create a new Markdown formatted file in the `drafts` folder.
+- Collect feedback/iterate on the draft, and merge it after 24/48 hours.
```

Comment:
I would make a hard deadline; otherwise, it's going to be hard to know when to merge. A hard deadline puts the onus on others to comment, not on the writer of the draft to go around asking people to review it.